### PR TITLE
Type package for the npm unix-time package

### PR DESCRIPTION
**Pull request by b1z3rr4**
# Type package for the npm unix-time package
**This is a branch that aims to add a new type package that references the unix-time npm package**

**I'm not the creator of the original npm package, but I use it frequently and in case of code changes or deletions, I'll be careful to update here.**

**Find it at: [Package unix-time](https://www.npmjs.com/package/unix-time)
Me: [b1z3rr4](https://github.com/b1z3rr4)**

### CODE REVIEW

File: types/unix-time/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+declare const unixTime: UnixTime
```
Comment (peterblazejewicz) [inline types/unix-time/index.d.ts:1]: when using scaffold code with templates for module, all the 'docu' like stuff is added (header file with version/author/reference).
There wouldn't be required to add all those excusions.

**Comment by peterblazejewicz** [comment]
The `LICENSE` file to be shipped with package would probably require to be added in additional file: 'OTHER_FIILES':
https://github.com/DefinitelyTyped/DefinitelyTyped#other_filestxt
I don't recall personally DT package shipping it's own license, to be checked

File: types/unix-time/.eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+    "rules": {
```
Comment (sheetalkamat) [inline types/unix-time/.eslintrc.json:2]: remove this file

File: types/unix-time/tslint.json [hunk 3]
```diff
@@ -0,0 +1,8 @@
+{
+    "extends": "@definitelytyped/dtslint/dtslint.json",
+    "rules": {
+        "no-var-keyword": false,
+        "prefer-const": false,
+        "semicolon": false
+    }
```
Comment (sheetalkamat) [inline types/unix-time/tslint.json:7]:
```suggestion
```
Fix errors after removing the rules

File: types/unix-time/.eslintrc.json [hunk 4]
```diff
@@ -0,0 +1,6 @@
...
+        "dt-header": "off",
+        "trim-file": "off"
+    }
```
Comment (sheetalkamat) [inline types/unix-time/.eslintrc.json:5]: Please use template as suggested at https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package so you arent adding unnecessary files.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]
Fix the CI errors

**Pull request closed without merging**